conf/common.py

```diff
@@ -534,3 +534,22 @@ def parse_nvme( dev_name ):
     if 'p' in dev_name:
         dev_name = dev_name.split('p')[0]
     return dev_name
+
+def parse_device_name(dev_name):
+    sata_pattern = re.compile(r'^sd\w*$')
+    nvme_pattern = re.compile(r'^nvme\w*$')
+
+    if sata_pattern.match(dev_name):
+        if re.compile(r'^sd\D\d*$').match(dev_name):
```

Comment:
should we consider multi \D here? if dev name is /dev/sdab? seems we just need to add * on \D?